docs/doc/10-deploy/00-understanding-deployment-modes.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Understanding Deployment Modes
+sidebar_label: Understanding Deployment Modes
+description:
+  Describes Databend deployment modes
+---
+
+## Understanding Deployment Modes
+
+A Databend deployment includes two types of node, Meta and Query. The Meta node stores various types of metadata (such as database, table, cluster, and transaction) and manages user information (including authorization and authentication). The Query node takes care of queries. 
+
+When deploying Databend, you specify a deployment mode, standalone, or cluster. A standalone Databend allows for one Meta node and one Query node, and a Databend cluster can include multiple Meta and Query nodes to enhance the computing capability.
+
+### Compatible Object Storage Solutions
+Databend works with self-hosted or cloud object storage solutions. You will need to prepare your own object storage before deploying Databend. The following is a list of compatible object storage solutions:
+- Self-hosted object storage solutions:
+  - MinIO
+  - SeaweedFS
+- S3 compatible storage solutions:
+  - Amazon S3
+  - Tencent COS
+  - Alibaba OSS
+  - Wasabi
+  - QingCloud QingStore
+- Azure Blob Storage
```

Comment:
The relations between them, IMHO, are wrong.

`Self-hosted object storage solutions` listed here are all `S3 compatible storage solutions`.